Set-1/Solved Programs/BrainCar/TestCaseForBrainCar.java

```diff
@@ -0,0 +1,15 @@
+package com.braincar;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+
+public class TestCaseForBrainCar {
+
+	@Test
+	public  void testBrainCarName() {

```

Comment:
Can we add more unit test? to have better code coverage?
Also include negative test cases